terraform-registry-manifest.json

```diff
@@ -0,0 +1,6 @@
+{
+    "version": 1,
+    "metadata": {
+        "protocol_versions": ["5.0"]
```

Comment:
I'm not sure how precise or exhaustive this list has to be, but the manifest provider uses protocol, `5.2`
Unsure if we need to mention that in there. I'll check with the registry folks.

Reply:
Seems like we only need to specify major versions.
5.0 should be fine for us.